packages/good-design/src/core/modals/BasicModal.tsx

```diff
@@ -57,8 +56,8 @@ const BasicModal: FC<BasicModalProps> = ({
   return (
     /* height 100vh is required so modal always shows in the middle */
     <NBModal isOpen={modalVisible} onClose={onClose} {..._modal} minH="100vh" bgColor={bgOverlay}>
-      <Box bgColor={bgContent} borderRadius="lg" h={height} width={width}>
-        <NBModal.Content w={"100%"}>
+      <Box borderRadius="lg" h={height} width={width} bgColor="white">
```

Comment:
add prop for background, set it to white in the theme, re-use everywhere in this component

Reply:
withTheme hook? or just add a bgColor with default white as prop?

Reply:
done